apps/test/unit/templates/lessonOverview/activities/LessonTipTest.jsx

```diff
@@ -22,4 +22,17 @@ describe('LessonTip', () => {
     const safeMarkdown = wrapper.find('SafeMarkdown').first();
     expect(safeMarkdown.props().markdown).to.contain('Teaching tip content');
   });
+
```

Comment:
These look like good tests for LessonTip. I'm wondering if there should also be tests added to ActivitySection to check that tips start open?

Reply:
I tried this but I would have to change `shallow` to `mount` and that opened a whole redux can of worms. Since the logic is pretty encapsulated in `LessonTip` (for now), I think it's okay to go without a test here

When we add the "Expand/Collapse All" button, the state will get pulled up so I'm just kicking that can down the road

Reply:
Works for me! I know about that can of worms well....